src/Symfony/Bundle/FrameworkBundle/CacheWarmer/ValidatorCacheWarmer.php

```diff
@@ -79,8 +79,12 @@ protected function doWarmUp($cacheDir, ArrayAdapter $arrayAdapter)
 
     protected function warmUpPhpArrayAdapter(PhpArrayAdapter $phpArrayAdapter, array $values)
     {
+        $values = array_filter($values, function ($val) {
+            return $val !== null;
+        });
+
         // make sure we don't cache null values
-        parent::warmUpPhpArrayAdapter($phpArrayAdapter, array_filter($values));
+        parent::warmUpPhpArrayAdapter($phpArrayAdapter, $values);
```

Comment:
`array_filter($values)` will also remove everything that is `empty`.

https://3v4l.org/0m3In

There is no test for this code and I am not sure why `null` should be removed. I can only assume it is because of the callback =)